tsconfig.json

```diff
@@ -3,7 +3,7 @@
 		/* Visit https://aka.ms/tsconfig.json to read more about this file */
 		/* Basic Options */
 		// "incremental": true,                         /* Enable incremental compilation */
-		"target": "es5", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+		"target": "es6", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
Est-ce que c'était important qu'on passe en es6 ?

Reply:
Plus ou moins, c'est ce qui nous permet de faire le `for let of` dans [le grade module](https://github.com/TableauBits/Kalimba/pull/40/files#diff-b84fb3de80ef51ca214ab701468c2a52ce74a42ecddea0d288bdfa5b0d2f4386R94), mais y'aurais probablement moyen de faire sans si ça pose problème.